tests/test_servo.py

```diff
@@ -307,30 +315,59 @@ def test_load_configuration_to_subnode_zero(read_config, pytestconfig, connect_t
 
 @pytest.mark.canopen
 @pytest.mark.ethernet
-def test_store_parameters(connect_to_slave):
+@pytest.mark.ethercat
+def test_store_parameters(connect_to_slave, request):
+    user_over_voltage_register = "DRV_PROT_USER_OVER_VOLT"
+
     servo, net = connect_to_slave
-    assert servo is not None and net is not None
+
+    initial_user_over_voltage_value = servo.read(user_over_voltage_register)
```

Comment:
Does connect_to_slave load the config? So if we run this test N times the register does not continuously increase on each iteration?

Reply:
No, it does not load the config. The initial value is restored at the end of the test, it will not change if run N times.